app/app/components/settings/user-list.js

```diff
@@ -112,15 +113,15 @@ export default Ember.Component.extend({
 			let user = this.get('editUser');
 			let password = this.get('password');
 
-			if (is.empty(user.get('firstname'))) {
+			if (is.empty(user.firstname)) {
```

Comment:
You should use `user.get('firstname')` here. Same for all other places in this file.